communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -82,6 +85,7 @@
             GraphExport export = mod.graph(req);
 
             if (export == null || export.getRelationships() == null) {
+                // no dependencies found
```

Comment:
+1 To return empty GAVDependencyTree. When the export is null we should probably throw a RuntimeException (or is there any case, when it should be null?).